### PR TITLE
Minor adjustements

### PR DESCRIPTION
**Pull request by JAORMX**
Just some stylistic changes that I came up with while playing around with the code.

### CODE REVIEW

File: manifests/scripts.pp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Class: certmonger::scripts
+class certmonger::scripts {
+  $verifyscript = '/etc/ipa/verify_certmonger_request.sh'
```
Comment (earsdown) [inline manifests/scripts.pp]: Should we make this a class parameter?

**Comment by earsdown** [comment]
@JAORMX thanks for the contribution! Let me know your thoughts on my in-line comments.

File: manifests/scripts.pp (hunk 1 above)
Comment (JAORMX) [inline manifests/scripts.pp]: Actually I like the idea.  I'll do that.

Comment (JAORMX) [inline manifests/scripts.pp]: Done

**Pull request merged**